lib/utils/noFilesFoundError.js

```diff
@@ -11,9 +13,36 @@ class NoFilesFoundError extends Error {
 			fileList = [fileList];
 		}
 
-		const pattern = fileList.filter((i) => !i.startsWith('!')).join(', ');
+		const filteredFileList = fileList.filter((i) => !i.startsWith('!'));
+		const pattern = filteredFileList.join(', ');
 
 		this.message = `No files matching the pattern "${pattern}" were found.`;
+
+		if (
+			os.platform() === 'win32' &&
+			filteredFileList.filter((file) => file.match(/^'.*'$/)).length > 0
+		) {
+			const packageScript = process.env.npm_lifecycle_event || '';
+			const scriptContents = process.env.npm_lifecycle_script || '';
+
+			if (packageScript) {
+				this.message += `
+
+Please use escaped double quotes for a file pattern in your package.json:
+
+        "scripts": {
+                "${packageScript}": "${scriptContents.replace(/'([^']*)'/g, '\\"$1\\"')}"
+        },
```

Comment:
These 3 lines are using spaces instead of tabs

Reply:
Updated, thank you!